modules/end-user-portal/src/Program.cs

```diff
@@ -34,7 +35,7 @@ public static void Main(string[] args)
             .ConfigureWebHostDefaults(webBuilder =>
             {
                 webBuilder.UseStartup<Startup>();
-            })
+            }).UseServiceProviderFactory(new AutofacServiceProviderFactory())
```

Comment:
As mentioned above adding this for one class seems overkill

Reply:
It's now removed as no longer needed